docs/behaviors/testing-behaviors.rst

```diff
@@ -173,6 +173,74 @@ To run the test, we can use the usual test runner:
 
     $ ./bin/instance test -s collective.gtags
 
+Testing a dexterity type with a behavior
+----------------------------------------
+
+Lets say you want to test your dexterity type when a behavior is enabled or disabled, note that not all behaviors are enabled by default. To do this you will need to setup the behavior in your test. There is an example of this kind of test in the collective.cover product. There is a behavior that adds the capability for the cover page to refresh itself. The test check to see if the behavior effect on the page is not present, then enables the behavior, check its effect and then disables it again. Here is the code:
```

Comment:
May you break the line after each sentence? 
This makes a translators life easier and is also what our documentation style guide recommends.